### PR TITLE
Feature/us6972 microclient phoenix

### PR DESCRIPTION
**Pull request by richiebrees**
Routing merged, plus some updates to App.js.

### CODE REVIEW

File: apps/crossroads_interface/web/controllers/legacy_controller.ex [hunk 1]
```diff
@@ -1,14 +1,20 @@
 defmodule CrossroadsInterface.LegacyController do
   use CrossroadsInterface.Web, :controller
+  require IEx
```
Comment (mjvcallibrity) [inline apps/crossroads_interface/web/controllers/legacy_controller.ex:3]: Why do we need to require this?

File: apps/crossroads_interface/web/controllers/notfound_controller.ex [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    end
+    conn
+    |> render("404.html", %{ payload: payload, 
+      "css_files": [
+        "/js/legacy/core.css"
+      ]
+    })
+  end
+
+end
```
Comment (mjvcallibrity) [inline apps/crossroads_interface/web/controllers/notfound_controller.ex:21]: Time to write a test for this controller! :D

File: apps/crossroads_interface/web/views/error_view.ex [hunk 3]
```diff
@@ -20,7 +20,13 @@ defmodule CrossroadsInterface.ErrorView do
     #end
     # Fall through to the legacy crds-angular appliction and let
     # it handle the route.
-    render("index.html", assigns)
+    #conn = assigns[:conn]
+    #payload = case Pages.get_page("/servererror/", false) do
+      #{:ok, 200, body} -> Enum.at(body["pages"], 0)["content"]
+      #{_, _, body} -> "<h2> #{body} </h2>"
+    #end
+    IEx.pry()
```
Comment (mjvcallibrity) [inline apps/crossroads_interface/web/views/error_view.ex:28]: Do we need to keep the commented code and the IEx.pry?

File: apps/crossroads_interface/web/views/notfound_view.ex [hunk 4]
```diff
@@ -0,0 +1,3 @@
+defmodule CrossroadsInterface.NotfoundView do
+  use CrossroadsInterface.Web, :view
+end
```
Comment (mjvcallibrity) [inline apps/crossroads_interface/web/views/notfound_view.ex:3]: Time to add a view test! :D

File: apps/crossroads_interface/package.json [hunk 5]
```diff
@@ -1,9 +1,15 @@
 {
+    "scripts": {
+        "build-prod": "./node_modules/brunch/bin/brunch build --production",
+        "watch": "./node_modules/brunch/bin/brunch watch --stdin",
+        "build": "./node_modules/brunch/bin/brunch build"
+    },
     "repository": {},
     "dependencies": {
+        "crds-styles": "crdschurch/crds-styles#development",
+        "handlebars": "^4.0.6",
```
Comment (richardbarker) [inline apps/crossroads_interface/package.json]: Can we remove this?

File: apps/crossroads_interface/web/static/js/app.js [hunk 6]
```diff
@@ -1,21 +1,43 @@
...
+            event.preventDefault();
+            alert("Hello from Phoenix JavaScript!");
+        });
+    }
+}
+
+function translateBaseHref() {
+    // This could be used to replace {{ }} variables in the nav menu
+    // FWIW I don't like it!
+    var source = document.getElementById("nav-menu");
```
Comment (richardbarker) [inline apps/crossroads_interface/web/static/js/app.js]: Does CMS change make this unnecessary?

**Comment by richiebrees** [comment]
Merged to the new *CRDS-MAESTRO* repo -- closing.

**Pull request closed without merging**